docs/develop/convert-javascript-to-typescript.md

```diff
@@ -43,6 +32,41 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 ## Convert the add-in project to TypeScript
 
+### Add Nuget packages
+
+1. Open the Nuget package manager by choosing **Tools > Nuget Package Manager > Manage Nuget Packages for Solution**
+1. Select the **Browse** tab and search for **Microsoft.TypeScript.MSBuild**. Install this package to the ASP.NET web project, or update it if it's already installed. This will ensure the project will transpile to JavaScript when the build runs.
+1. On the **Browse** tab, search for  **jquery.TypeScript.DefinitelyTyped**. Install this package to the ASP.NET web project, or update it if it's already installed. This will ensure the jQuery TypeScript definitions are included in your project. The packages for jQuery appear in a file generated by Visual Studio, called **packages.config**.
+
+> [!NOTE]
+> In your TypeScript project, you can have a mix of TypeScript and JavaScript files and your project will compile. This is because TypeScript is a typed superset of JavaScript that compiles JavaScript.
+
+### Create a TypeScript config file
+
+1. In **Solution Explorer** create a new file named **tsconfig.json** under the web project. The web project has the name you created the project with and the text **Web** appended to the name.
+1. Enter the following JSON into the **tsconfig.json** file.
+
+```json
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "noImplicitAny": false,
+    "noEmitOnError": true,
+    "removeComments": false,
+    "sourceMap": true,
+    "target": "es5"
+  },
+  "include": [
+    "scripts/**/*",
+    "**/*"
+  ]
+}
+```
+
+1. Save the file. For more information on **tsconfig.json** settings, see [What is a tsconfig.json?](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html)
+
+### Update the JavaScript files
+
```

Comment:
Consider adding a lead-in sentence to cover projects that weren't just created. Maybe something like, "Change your JavaScript files (**.js**) to TypeScript files (**.ts**). Then, make the necessary changes for them to compile. This section walks through the default files in a new project."

Reply:
Nice, added.